app/blocks/input/prefixScript.js

```diff
@@ -33,16 +33,25 @@ class PrefixScript extends InputBlock {
       return false
     }
     var regex = ['^']
-    if (!this.isScoped) {
+    if (this.isScoped) {
+      if (this.args.match(/^r/i)) {
```

Comment:
Is there a reason behind using `/^r/` instead of `/^required/`? 

Is it to protect against people misspelling the word 'Required' ?